hazelcast/src/main/java/com/hazelcast/config/alto/AltoConfig.java

```diff
@@ -25,9 +22,8 @@ public int getEventloopCount() {
     }
 
     public AltoConfig setEventloopCount(int eventloopCount) {
-        if (eventloopCount < Bounds.MIN_EVENTLOOP_COUNT || eventloopCount > Bounds.MAX_EVENTLOOP_COUNT) {
-            throw new InvalidConfigurationException("Buffer size should be between "
-                    + Bounds.MIN_EVENTLOOP_COUNT + " and " + Bounds.MAX_EVENTLOOP_COUNT);
+        if (eventloopCount < 1) {
```

Comment:
Are we not using Preconditions any longer?

Reply:
https://github.com/hazelcast/hazelcast/pull/23319/commits/16aaf911c270da78592443f9d81c145858b9459e